auter.yumdnfModule

```diff
@@ -87,7 +87,12 @@ function prepare_updates() {
         logit "INFO: No updates are available to be downloaded."
       fi
     else
-      logit "WARNING: downloadonly option is not available"
+      if [[ "${ONLYINSTALLFROMPREP}" == "yes" ]]; then
```

Comment:
This is already nested within the outer `if [[ "${PREDOWNLOADUPDATES}" == "yes" ]]; ...` on  line 48 so will always evaluate to true. Might as well remove the `else`?

The same goes for `auter.aptModule`.